cmd/containerd/command/main.go

```diff
@@ -215,6 +239,9 @@ func applyFlags(context *cli.Context, config *srvconfig.Config) error {
 			*v.d = s
 		}
 	}
+
+	applyPlatformFlags(context)
```

Comment:
I don't understand how this can happen so late. It seems to set the global flags and it seems this needs to happen before calling for example, `registerUnregisterService`. What am I missing?

Reply:
It is called before `registerUnregisterService` 

```
		// Apply flags to the config
		if err := applyFlags(context, config); err != nil {
			return err
		}

		// Make sure top-level directories are created early.
		if err := server.CreateTopLevelDirectories(config); err != nil {
			return err
		}

		// Stop if we are registering or unregistering against Windows SCM.
		stop, err := registerUnregisterService(config.Root)
		if err != nil {
			logrus.Fatal(err)
		}
		if stop {
			return nil
		}
```